eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/rebalance/EventmeshRebalanceImpl.java

```diff
@@ -62,7 +62,7 @@ public void doRebalance() throws Exception {
         final String cluster = eventMeshTCPServer.getEventMeshTCPConfiguration().getEventMeshCluster();
         //get eventmesh of local idc
         Map<String, String> localEventMeshMap = queryLocalEventMeshMap(cluster);
-        if (localEventMeshMap == null || localEventMeshMap.size() == 0) {
+        if (localEventMeshMap == null || localEventMeshMap.isEmpty()) {
```

Comment:
use MapUtils.isEmpty()(https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/MapUtils.html) is better.